zk-compression-docs/CLAUDE.md

```
@@ -274,12 +274,15 @@ Resources
+```text
+References
+  ├── Whitepaper (references/whitepaper.md)
+  │   └── PDF: <https://github.com/Lightprotocol/light-protocol/blob/main/light_paper_v0.1.0.pdf>
+  ├── Node Operators (references/node-operators.md)
+  │   ├── Photon indexer: <https://github.com/helius-labs/photon>
+  │   ├── Prover: <https://github.com/Lightprotocol/light-protocol/tree/main/light-prover>
+  │   └── Forester: <https://github.com/Lightprotocol/light-protocol/tree/main/forester>
+  ├── Terminology (references/terminology.md)
+  └── Security (references/security.md)
+      └── audits: <https://github.com/Lightprotocol/light-protocol/tree/main/audits>
+```
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)